cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2652,7 +2655,7 @@ String getCertificateThumbprint(Secret certSecret, CertAndKeySecretSource custom
         }
 
         Future<ReconciliationState> kafkaBrokersSecret() {
-            return updateCertificateSecretWithDiff(KafkaCluster.brokersSecretName(name), kafkaCluster.generateBrokersSecret())
+            return updateCertificateSecretWithDiff(KafkaCluster.brokersSecretName(name), kafkaCluster.generateBrokersSecret(clusterCa, clientsCa))
```

Comment:
Why are you not changing it also for the other compoenents? UO, TO, KE, CC, ...

Reply:
So ... UO, TO and KE uses `ModeUtils.buildSecret` to build their Secrets and I updated that. Good point about CC and the others I will check.